proto/Makefile

```diff
@@ -16,7 +16,7 @@
 # Author: Spencer Kimball (spencer.kimball@gmail.com)
 # Author: Andrew Bonventre (andybons@gmail.com)
 
-PROTOC      := ../_vendor/usr/bin/protoc
+PROTOC      := protoc
```

Comment:
Ah, cool. We should refer to the protoc built in this way with `../../../../../bin/` as we do for protoc-gen-gogo below. That way we don't require `$GOPATH/bin` to be in `$PATH`.